driver-core/src/main/com/mongodb/internal/operation/Operations.java

```diff
@@ -220,21 +221,22 @@ public EstimatedDocumentCountOperation estimatedDocumentCount(final EstimatedDoc
                 .batchSize(batchSize)
                 .collation(collation)
                 .hint(hint == null ? null : hint.toBsonDocument(documentClass, codecRegistry))
-                .comment(comment);
-
+                .comment(comment)
+                .let(variables == null ? null : variables.toBsonDocument(documentClass, codecRegistry));
```

Comment:
Use the: `com.mongodb.internal.operation.Operations#toBsonDocument` helper

Reply:
Done